dyn_em/module_big_step_utilities_em.F

```diff
@@ -2687,7 +2672,7 @@ SUBROUTINE w_damp( rw_tend, max_vert_cfl,max_horiz_cfl, &
 ! This internal write is costly on newer Xeon processors because it breaks
 ! vectorization.  (J. Michalakes for L. Meadows at Intel, 12/13/2012)
 
-         IF (vert_cfl .gt. w_crit_cfl) THEN
```

Comment:
Maybe we can use w_crit_cfl when ieva is on and 2 when it is off. This could be done where w_beta is used above with another local variable like cflprint. Line in print would also need modifying.

Reply:
@dudhia That is assuming someone would set w_crit_cfl other than 2?

Reply:
Mainly to keep the ieva print as it was. Perhaps Lou wants that threshold shown when ieva is on.

Reply:
@dudhia Lou recommended using w_crit_cfl = 2 when turning on ieva, so 2 seems to be good criterion for printing.

Reply:
@dudhia w_crit_cfl can be set in the namelist, and the recommended value is 2 if ieva is on. This value happens to be the value we turn on CFL prints in the past. Another way to fix this partially is to change the registry value of w_crit_cfl from 1.2 to 1. This will turn on w_damp at this value, but print criterion still needs to be changed.